src/cityreader/cityreader.py

```diff
@@ -21,7 +28,12 @@ def cityreader(cities=[]):
   # Ensure that the lat and lon valuse are all floats
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+    with open("cities.csv", newline='') as city_file:
```

Comment:
Can you think of a way to make this `open()` function less fragile? 

(Hint: see my previous review note on https://github.com/thisbenrogers/Intro-Python-I/pull/1#discussion_r453169610)

Reply:
I did initially implement the `pathlib` parent path, but I couldn't get my (prewritten) tests to pass. I pulled it out and passed the tests. Will absolutely be digging further into `pathlib` however.

Reply:
@thisbenrogers totally makes sense! Took me a little bit to figure out how to grok `pathlib` / `os.path` and use it effectively. 

I wonder if the prewritten tests were writen in an odd way that is tightly coupled to the implementation? Interesting. 🤔